addon-test-support/helpers/set-runtime-config.js

```diff
@@ -0,0 +1,18 @@
+import { registerDestructor } from '@ember/destroyable';
+import { getContext } from '@ember/test-helpers';
+
+const initialConfig = Symbol('setRuntimeConfig');
+
+export function setRuntimeConfig(config) {
+  const context = getContext();
+
+  if (!context[initialConfig]) {
+    context[initialConfig] = { ...window._erc };
```

Comment:
@anbykova why do we need spread here? can't we keep the reference to the original object and restore it?

Reply:
@SergeAstapov  First, we can't do ```window._erc = ...``` because in that case window._erc and config (```import config from 'ember-runtime-config'```) would lead to different objects.

So we need to rewrite properties with ```Object.assign```. 
If we don't copy ```window._erc``` with spread operator we rewrite properties of the original object at the next step.

Reply:
@anbykova ah makes sense. Thanks!